src/gui/newwizard/states/oauthcredentialssetupwizardstate.cpp

```diff
@@ -24,11 +25,25 @@ namespace OCC::Wizard {
 OAuthCredentialsSetupWizardState::OAuthCredentialsSetupWizardState(SetupWizardContext *context)
     : AbstractSetupWizardState(context)
 {
-    auto oAuthCredentialsPage = new OAuthCredentialsSetupWizardPage(_context->accountBuilder().serverUrl());
+    auto modernWebFingerAuthenticationServerUrl = _context->accountBuilder().modernWebFingerAuthenticationServerUrl();
+
+    auto oAuthCredentialsPage = [&]() {
```

Comment:
See the comment in https://github.com/owncloud/client/pull/10665/files#diff-b5743c10075cf6d5ad7b5e977596a1f952559ff91c05d950f9108ab7e60354b1R65